machines/woodpecker-server/configuration.nix

```diff
@@ -17,13 +17,19 @@
   pinpox.services.openssh.enable = true;
 
   lounge-rocks = {
-    hetzner = {
+    cloud-provider = {
       enable = true;
-      interface = "enp1s0";
-      ipv6_address = "2a01:4f8:1c17:636f::";
+      hetzner = {
+        enable = true;
+        interface = "enp1s0";
+        ipv6_address = "2a01:4f8:1c17:636f::";
+      };
     };
     nix-common.enable = true;
-    attic.server.enable = true;
+    attic.server = {
+      enable = true;
+      enableNginx = true;
```

Comment:
Not really a fan of putting nginx proxies into the modules itself, as it limits composability and doesn't allow other proxies. Also consider, we might not want to hard-code the port and listening address.

Reply:
That's a valid point!
I will remove nginx from the module once we deployed everything.
Current plan: reverse proxy won't be on the same server anyway - so we can completely remove it.